cahee/css/styles_cahee_header.css

```diff
@@ -83,3 +91,104 @@
     margin-left: 1em;
   }
 }
+
+@media screen and (max-width: 767px) {
+  #nav-container {
```

Comment:
You are using both id and class for that element (`.nav-container` and `#nav-container`). Does it need to be so? We can just use` .nav-container` selector without increasing specificity since you'd used it already

Reply:
It has to be with increased specificity, i.e. with # otherwise styles will be overwritten and the side menu will be not shown. I tried to change to use selector by class but it didn't work.